compiler/include/llvmGlobalToWide.h

```diff
@@ -33,6 +33,7 @@ namespace llvm {
 
 #include "llvm/ADT/DenseMap.h"
 #include "llvm/IR/ValueHandle.h"
+#include <vector>
```

Comment:
I was expecting there to be code changes in this file along with the additional include, was this an accidental commit or is LLVM 13 more sensitive to having forgotten it?

Reply:
I think either `"llvm/ADT/DenseMap.h"` or `"llvm/IR/ValueHandle.h"` used to include `<vector>`, but don't any more in LLVM 13. That lead to an error for not having the definition of vector at line 142.

```
In file included from llvmGlobalToWide.cpp:37:
../include/llvmGlobalToWide.h:142:22: error: implicit instantiation of undefined
      template 'std::__1::vector<llvm::WeakVH, std::__1::allocator<llvm::WeakVH>
      >'
  specialFunctions_t specialFunctions;
                     ^
/Library/Developer/CommandLineTools/usr/bin/../include/c++/v1/iosfwd:216:28: note:
      template is declared here
class _LIBCPP_TEMPLATE_VIS vector;
                           ^
1 error generated.
```

Reply:
Ah, neato.  Thanks!